pages/api/mail/index.ts

```diff
@@ -0,0 +1,63 @@
+import { ResponseUtil } from "@utils/responseUtil";
+import type { NextApiRequest, NextApiResponse } from "next";
+import send from "@utils/sendMail";
+import findEmails from "@database/mail";
+
+/**
+ * Handles the request to /api/mail (POST)
+ * @param req request object
+ * @param res response object
+ * @returns Promise<void> and sends json object back to the client (type: ResponseObject)
+ */
+export default async function mailHandler(
+    req: NextApiRequest,
+    res: NextApiResponse,
+): Promise<void> {
+    if (req.method == "POST" && req.body.key == process.env.AWS_LAMBDA_KEY) {
+        // AWS_LAMBDA_SECRET
+        const result3h = await findEmails(3);
```

Comment:
Make 3 a constant

Reply:
Also, `result3h` isn't really descriptive. What you're returning is a list of classes joined together w other stuff right?